src/command/cmd_defs.c

```diff
@@ -1300,7 +1300,7 @@ static const struct cmd_t command_defs[] = {
       CMD_ARGS(
               { "maxtabs <value>", "Set the maximum number of tabs to display, <value> must be between 0 and 10." },
               { "tablen <value>", "Set the maximum number of characters to show as the tab name, 0 sets to unlimited." },
-              { "tabmode default|actlist", "Set the mode how the 'active tabs' are shown." },
+              { "tabmode static|dynamic|actlist", "Set the mode tabs are shown. `dynamic` is a mode that modernizes paging by displaying tabs around current tab. `actlist` setting shows only active tabs. `static` setting always shows tabs in 1 to max_tabs range." },
```

Comment:
Maybe describe only what it does. Remove `is a mode that modernizes paging by` displaying-> displays.